### ISSUE TITLE
Having conflict issues with rubocop 1.3 and solargraph 0.39.17

### ISSUE THREAD

**Issue by LucasArruda** [castwide/solargraph#391]
I am recently trying to upgrade to ruby 2.7 from 2.6.

With that, I also updated a few gems, including `rubocop`.

But now I am getting this error when I try to bundle

```bash
Bundler could not find compatible versions for gem "rubocop":
  In Gemfile:
    rubocop (~> 1.3)

    solargraph (~> 0.39) was resolved to 0.39.17, which depends on
      rubocop (~> 0.52)
```

I wonder if `rubocop` 1.3 is incompatible with `solargraph`?

Is bundler messing this up?

Or this gem should do `rubocop (>= 0.52)` instead of `rubocop (~> 0.52)`?

Here is all my `rubocop` related gems:

```
  gem "guard-rubocop", "~> 1.4", require: false
  gem 'rubocop-rails', "~> 2.8", require: false
  gem "rubocop-rails_config", "~> 1.0"
  gem 'rubocop-rspec', "~> 1.38", require: false
```

Many thanks!

**Comment by castwide** [comment]
This should be fixed by #381 in an upcoming release. There are a couple of technical issues that need resolved, but it's a priority.

**Comment by krnjn** [comment]
Any update on this?

**Comment by castwide** [comment]
#381 is merged into master, and there's a v0.39.18 branch to release the update, but the tests are failing in Travis CI. I'm looking into it.

**Comment by numbcoder** [comment]
@castwide When will the new version be released?

**Comment by castwide** [comment]
@numbcoder It'll be released in the next day or so. The failing tests after #381 are resolved by #394.

**Comment by castwide** [comment]
Released in version 0.40.0.

**Comment by LucasArruda** [comment]
Thanks @castwide . Working now!
